camel/model_backend.py

```diff
@@ -66,7 +67,7 @@ def run(self, *args, **kwargs) -> Dict[str, Any]:
         num_max_token = num_max_token_map[self.model_type.value]
         num_max_completion_tokens = num_max_token - num_prompt_tokens
         self.model_config_dict['max_tokens'] = num_max_completion_tokens
-        response = openai.ChatCompletion.create(*args, **kwargs,
+        response = litellm.completion(*args, **kwargs,
```

Comment:
When testing this for Claude, line 78 below ("if not isinstance(response, Dict)") fails because the response is an instance of ModelResponse. Similar thing happening in chat_agent.py line 192.